lldb-eval/context.cc

```diff
@@ -224,7 +229,12 @@ std::unique_ptr<ParserContext::IdentifierInfo> Context::LookupIdentifier(
   // Lookup context variables first.
   auto context_var = context_vars_.find(name);
   if (context_var != context_vars_.end()) {
-    return IdentifierInfo::FromContextVar(context_var->second);
+    return IdentifierInfo::FromValue(context_var->second.inner_value());
+  }
+
+  auto context_arg = context_args_.find(name);
+  if (context_arg != context_args_.end()) {
+    return IdentifierInfo::FromContextVar(context_arg->second);
```

Comment:
Maybe rename it to `FromContextArg` for consistency with other namings?
Or maybe even better to `FromType` for symmetry with `FromValue`.

Reply:
Replaced with `FromContextArg` (to avoid confusion since there already is `FromThisKeyword` which takes a single type as parameter).